docs/Notes.puml

```diff
@@ -0,0 +1,172 @@
+//@@author WilsonLee2000
+
+@startuml
+title Sequence Diagram for Notes Functionalities
+'https://plantuml.com/sequence-diagram
+
+actor User
+
+User -> BadMaths : main()
+activate BadMaths
+BadMaths -> NotesList: NotesList(Storage.loadFile(path))
+NotesList -> Storage : loadFile(String path)
+activate Storage
+deactivate Storage
+activate NotesList
+NotesList --> BadMaths
+deactivate NotesList
+BadMaths -> Parser : Parser(userInput)
+activate Parser
+Parser --> BadMaths : getCommand()
+Parser --> BadMaths : getToDo()
+deactivate Parser
+alt inputCommand is null
+activate Command
+    BadMaths -> Command : Command(command, toDo);
```

Comment:
Perhaps this method invocation arrow should be connected to the constructor activation bar instead rather than a regular method activation bar. Maybe try adding `create Command` to display the constructor activation bar earlier in the PUML file?